### PR TITLE
asadiqbal08/mitxpro-595+686 ZenDesk theme updates: round-2

### PR DESCRIPTION
**Pull request by asadiqbal08**
Task: [594](https://github.com/mitodl/mitxpro/issues/594)

fixes https://github.com/mitodl/mitxpro/issues/686
fixes https://github.com/mitodl/mitxpro/issues/594

#### Acceptance Criteria:

- [x] less space between buttons and contact us button 
- [x] remove "Contact Us" from the "Submit a Request" form (it just links to the same page) 
- [x] remove "Contact Us" from header on "Submit a Request" form (it just links to the same page) 
- [x] Submit button should be same blue as Contact Us button
- [x] "Submit a Request" header => "Submit a Request or Provide Feedback"
- [x]  MIT - > mit.edu
- [x]  xPRO -> xpro.mit.edu
- [x]  support center -> https://xpro.zendesk.com/hc/en-us

#### Screenshots: 

screencapture-xpro-zendesk-hc-en-us-2019-06-25-18_17_55

==============================

screencapture-xpro-zendesk-hc-en-us-requests-new-2019-06-25-18_01_39

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -166,3 +166,12 @@ document.addEventListener('DOMContentLoaded', function() {
     });
   }
 });
+
+$( document ).ready(function() {
+  $('a.submit-a-request').each(function () {
+    if (location.pathname.indexOf('/requests/new') > -1) {
```
Comment (asadiqbal08) [inline script.js:172]: With the help of jQuery is the only way to hide it.

File: templates/home_page.hbs [hunk 2]
```diff
@@ -80,4 +80,5 @@
       {{recent_activity}}
     {{/if}}
   </section>
+ -->
```
Comment (asadiqbal08) [inline templates/home_page.hbs:86]: yet I comment out that block. yet it was not the part of design.

**Comment by Ferdi** [comment]
The links in the top header (logo) should be 

MIT - > mit.edu
xPRO ->  xpro.mit.edu
support center -> https://xpro.zendesk.com/hc/en-us

**Comment by abdulkdawson** [comment]
@asadiqbal08 the Submit button text should be all Caps and bold.

**Comment by asadiqbal08** [comment]
@abdulkdawson @Ferdi you can view theme [here](https://xpro.zendesk.com/theming/theme/045300d5-1568-427c-b6c6-8bbe4e191a25)

good to merge ?

**Comment by pdpinch** [comment]
I don't think the font size on the submit page is correct:

image

**Comment by pdpinch** [comment]
This may be a little out of scope, but the "Contact Us" button should also be all caps. 

And @abdulkdawson is going to send you some new icons for the current categories.

**Comment by abdulkdawson** [comment]
@asadiqbal08 here are the two remaining icons for the support page. 

Payment-and-Refund

troubleshooting

**Comment by asadiqbal08** [comment]
@abdulkdawson these .png's are much larger in size -- `513x512`
The current one in repo have size of `77x79`

Can you please share a smaller ones ? Thanks.

**Comment by abdulkdawson** [comment]
@asadiqbal08 here are the resized icons.

troubleshooting
Payment-and-Refund

**Comment by asadiqbal08** [comment]
@abdulkdawson are we good now ? 

screencapture-xpro-zendesk-hc-en-us-2019-06-28-20_08_38

**Comment by abdulkdawson** [comment]
@asadiqbal08 yes this is good. Thank you!

**Pull request merged**